packages/react-databrowser/src/components/databrowser/hooks/use-fetch-simple-key.tsx

```diff
@@ -0,0 +1,27 @@
+import { useDatabrowser } from "@/store"
+import type { DataType } from "@/types"
+import { useQuery } from "@tanstack/react-query"
+
+import { useDeleteKeyCache } from "./use-delete-key-cache"
+
+export const FETCH_SIMPLE_KEY_QUERY_KEY = "fetch-simple-key"
+
+/** Simple key standing for string or json */
+export const useFetchSimpleKey = (dataKey: string, type: DataType) => {
+  const { redis } = useDatabrowser()
+  const { deleteKeyCache } = useDeleteKeyCache()
+
+  return useQuery({
+    queryKey: [FETCH_SIMPLE_KEY_QUERY_KEY, dataKey],
+    queryFn: async () => {
+      let result
+      if (type === "string") result = (await redis.get(dataKey)) as string | null
+      else if (type === "json") result = (await redis.json.get(dataKey)) as string | null
+      else throw new Error(`Invalid type when fetching simple key: ${type}`)
+
+      if (result === null) deleteKeyCache(dataKey)
+
+      return result
+    },
+  })
```

Comment:
We should maybe invalidate this query with an interval

Reply:
It was like that in the old version, i think it's a bit unexpected from some users so I removed all queries with intervals
https://linear.app/upstash/issue/DX-1303/data-browser-keeps-sending-requests-while-monitoring